docs/LoaderLayerInterface.md

```diff
@@ -1602,82 +1650,97 @@ Here's an example of a meta-layer manifest file:
-        by the supported extension.</td>
-    <td><small>vkEnumerateDeviceExtensionProperties</small></td>
+    <td>"layer"</td>
+    <td>The identifier used to group a single layer's information together.
+    </td>
+    <td>None</td>
+    <td>None</td>
+    <td><small>vkEnumerateInstanceLayerProperties</small></td>
   </tr>
   <tr>
-    <td>"enable_environment"</td>
-    <td><b>Implicit Layers Only</b> - <b>OPTIONAL:</b> Indicates an environment
-        variable used to enable the Implicit Layer (when defined to any
-         non-empty string value).<br/>
-        This environment variable (which may vary with each variation of the
-        layer) must be set to the given value or else the implicit layer is not
-        loaded.
-        This is for application environments (e.g. Steam) which want to enable a
-        layer(s) only for applications that they launch, and allows for
-        applications run outside of an application environment to not get that
-        implicit layer(s).</td>
-    <td><small>N/A</small></td>
+    <td>"layers"</td>
+    <td>The identifier used to group multiple layers' information together.
+        This requires a minimum Manifest file format version of 1.0.1.
+    </td>
+    <td>None</td>
+    <td>None</td>
+    <td><small>vkEnumerateInstanceLayerProperties</small></td>
   </tr>
   <tr>
-    <td>"disable_environment"</td>
-    <td><b>Implicit Layers Only</b> - <b>REQUIRED:</b> Indicates an environment
-        variable used to disable the Implicit Layer (when defined to any
-        non-empty string value).<br/>
-        In rare cases of an application not working with an implicit layer, the
-        application can set this environment variable (before calling Vulkan
-        functions) in order to "blacklist" the layer.
-        This environment variable (which may vary with each variation of the
-        layer) must be set (not particularly to any value).
-        If both the "enable_environment" and "disable_environment" variables are
-        set, the implicit layer is disabled.</td>
+    <td>"library_path"</td>
+    <td>Specifies either a filename, a relative pathname, or a full pathname to
+        a layer shared library file.
+        If "library_path" specifies a relative pathname, it is relative to the
+        path of the JSON manifest file (e.g. for cases when an application
+        provides a layer that is in the same folder hierarchy as the rest of the
+        application files).
+        If "library_path" specifies a filename, the library must live in the
+        system's shared object search path.
+        There are no rules about the name of the layer shared library files
+        other than it should end with the appropriate suffix (".DLL" on Windows,
+        ".so" on Linux, and ".dylib" on macOS).<br/>
+        <b>This field must not be present if "component_layers" is defined</b>.
+    </td>
+    <td>None</td>
```

Comment:
This probably should be "Explicit and Implicit Layers ONLY" (since it should only exist if its not a meta layer)

Reply:
I actually put "Not Valid for Meta-layers" instead because Meta-layers are a type of implicit layer.